router/router.nginx.conf.in

```diff
@@ -1,15 +1,3 @@
-server_names_hash_bucket_size 128;
-
-map $http_x_forwarded_proto $updated_scheme {
-     default $http_x_forwarded_proto;
-     '' $scheme;
-}
```

Comment:
Why did you delete these? Is it because they're in nginx.conf?

Reply:
Yes. I lifted all the `http` level directives into one place, including the new `access_log` and `log_format`.